public/static/locales/fr/login.json

```diff
@@ -0,0 +1,11 @@
+{
+    "title": "Sign in",
```

Comment:
Are we waiting for a proper translation or should we handle it ourselves?

Reply:
Waiting on translation.